### ISSUE TITLE
codechicken lib "Packet Leak Fix" causing crash on powered Draconic Chest open 

### ISSUE THREAD

**Issue by Krutoy242** [ACGaming/UniversalTweaks#541]
When `config/Universal Tweaks - Mod Integration.cfg/codechicken lib/Packet Leak Fix` is set to `true`, opening **powered** Draconic Chest causing crash:

```ruby
Description: Ticking player

io.netty.util.IllegalReferenceCountException: refCnt: 0
    at io.netty.buffer.AbstractByteBuf.ensureAccessible(AbstractByteBuf.java:1454)
    at io.netty.buffer.AbstractByteBuf.checkIndex(AbstractByteBuf.java:1383)
    at io.netty.buffer.UnpooledHeapByteBuf.copy(UnpooledHeapByteBuf.java:534)
    at io.netty.buffer.AbstractByteBuf.copy(AbstractByteBuf.java:1194)
    at codechicken.lib.packet.PacketCustom.copy(PacketCustom.java:773)
    at codechicken.lib.packet.PacketCustom.toPacket(PacketCustom.java:320)
    at codechicken.lib.packet.PacketCustom.sendToPlayer(PacketCustom.java:359)
    at com.brandon3055.brandonscore.lib.datamanager.TileDataManager.lambda$detectAndSendChangesToListeners$1(TileDataManager.java:90)
    at com.brandon3055.brandonscore.utils.DataUtils.forEachMatch(DataUtils.java:101)
    at com.brandon3055.brandonscore.lib.datamanager.TileDataManager.detectAndSendChangesToListeners(TileDataManager.java:90)
    at com.brandon3055.brandonscore.inventory.ContainerBCBase.detectAndSendChanges(ContainerBCBase.java:61)
    at net.minecraft.entity.player.EntityPlayerMP.TickCentral_TrueOnUpdateTick(EntityPlayerMP.java:324)
```

Disabling this config option fixed the issue.

[crash-2024-08-24_11.10.55-server.txt](https://github.com/user-attachments/files/16735945/crash-2024-08-24_11.10.55-server.txt)

![image](https://github.com/user-attachments/assets/c0b45fe4-d451-4e93-bca2-3bdf1364e8d1)

**Comment by ACGaming** [comment]
@jchung01

**Comment by jchung01** [comment]
Bug should be fixed next version with d1118c5c8, as you said the config option can be disabled for now.
